nuxeo-drive-core/src/test/java/org/nuxeo/drive/service/adapter/TestVersioningFileSystemItemFactory.java

```diff
@@ -682,15 +507,9 @@ public void testAutomaticVersioning() throws Exception {
     }
 
     @Test
-    @LocalDeploy({ "org.nuxeo.drive.core:OSGI-INF/test-nuxeodrive-versioningrules-contrib.xml",
-            "org.nuxeo.drive.core:OSGI-INF/test-nuxeodrive-configurationservice-contrib.xml" })
+    @LocalDeploy({ "org.nuxeo.drive.core:OSGI-INF/test-nuxeodrive-versioning-file-policy-contrib.xml" })
     public void testAutomaticVersioningAndDriveForceVersionDisabled() throws Exception {
 
-        // Check the property is correctly configured from the configuration
-        // service
-        ConfigurationService cs = Framework.getLocalService(ConfigurationService.class);
-        assertEquals("false", cs.getProperty(FileSystemItemHelper.NUXEO_DRIVE_FORCE_VERSIONING_PROPERTY));
-
         FileItem fileItem = (FileItem) customFileSystemItemFactory.getFileSystemItem(file);
 
         // As a user with READ permission
```

Comment:
Remove "in {@link DefaultFileSystemItemFactory#needsVersioning(DocumentModel)}" in `ensureJustModified`